services/coderabbit/coderabbit-stats.service.js

```diff
@@ -0,0 +1,69 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  reviews: Joi.number().required(),
+}).required()
+
+class CodeRabbitStats extends BaseJsonService {
+  static category = 'analysis'
+  static route = {
+    base: 'coderabbit',
+    pattern: 'stats/:provider/:org/:repo',
```

Comment:
Can we make the "noun" part of the route `/prs` or `/pull-requests` here. `/stats` doesn't really describe the information on the badge. It also leaves things more open if we ever wanted to add more CodeRabbit badges.
Docs on badge URLs: https://github.com/badges/shields/blob/master/doc/badge-urls.md

Reply:
@chris48s this would be a major change on the endpoint side as we need to change the backend code. We are on a freeze now and can make this change in Jan 2025.

Reply:
What we are declaring here is the URL for the badge on shields.io

The route on the upstream API (i.e: https://api.coderabbit.ai/stats/... ) is independent of this.

Reply:
Sorry for mis-understanding. Made the change